lib/ex_doc.ex

```diff
@@ -3,28 +3,49 @@ defmodule ExDoc do
   Elixir Documentation System. ExDoc produces documentation for Elixir projects
   """
 
+  defmodule Default do
+    @moduledoc false
+
+    @config %{
+      :formatter => "html",
+      :output => "./doc",
+      :source_ref => "master",
+      :retriever => ExDoc.Retriever,
+    }
+
+    @spec config :: map
+    def config,
+      do: @config
```

Comment:
I think we tend to avoid the `, do:` style when there's only one function clause and go with `do ... end`.